client/src/views/add-recipe.js

```diff
@@ -1,15 +1,42 @@
-import React from "react";
-import { Text, View, Button } from "react-native";
+import React, { useState } from "react";
+import { Text, View, Button,StyleSheet } from "react-native";
+import { TextInput } from "react-native-gesture-handler";
+import { set } from "react-native-reanimated";
 import { useBack } from "../service/utils/hooks/history";
 
 const AddRecipe = () => {
   const goBack = useBack();
+  const [recipe,setRecipe]=useState('');
+  const[recipeList,setRecipeList]=useState([]);
```

Comment:
The AddRecipe component should not contain nor display the recipe list. this should only be a form adding a recipe making it possible to add a recipe manually or store a reference URL. Please see mockups.